gradio/interface.py

```diff
@@ -546,7 +546,15 @@ def clean_html(raw_html):
                     for component in self.output_components:
                         component.render()
                     with Row():
-                        flag_btn = Button("Flag")
+                        if self.allow_flagging == "manual":
+                            flag_btn = Button("Flag")
+                            flag_btn._click_no_preprocess(
+                                lambda *flag_data: self.flagging_callback.flag(
+                                    flag_data
```

Comment:
Not sure about this, but wouldn't this be the correct version:

```suggestion
                                    *flag_data
```

Reply:
Nope `flagging_callback.flag()` takes a list for its first parameter, which is what `flag_data` is. `*flag_data` would unravel the list into separate parameters, which is not what we want here.